README.md

```diff
@@ -58,28 +124,27 @@ yarn prod-server
 pm2 start bin/pm2.yml
 ```
 
-## Build and deploy to AWS Lambda
+## Building and Deploying to AWS Lambda
 
-AWS Lamda with API Gateway and DynamoDB would give us a flexible way to deploy bot.
+AWS lamda with API Gateway and DynamoDB would give us a flexible way to deploy the bot.
 
-- **ONLY works in linux**, AWS Lambda is in Linux x64, some dependencies need to be prebuilt and upload to Lambda, so need the build process in Linux x64, you could do it in CI or any Linux server/destop env.
+*Be aware that AWS lamba **ONLY works in linux** on an x64 architecture. Therefore, some dependencies will need to be prebuilt and uploaded to lamda on a linux x64 instance. You could do this in ci or any linux server/destop env.*
```

Comment:
AWS Lambda is spelled `Lambda` with a capital `L` and a `b`.

https://aws.amazon.com/lambda/